### ISSUE TITLE
Ensure accessible target size for commonly used controls

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#7549]
The <span title="Web Content Accessibility Guidelines">WCAG</span> 2.1 <span title="Success Criterion">SC</span> [2.5.5 Target Size](https://www.w3.org/TR/WCAG21/#target-size) requires targets for [_pointer input_](https://www.w3.org/TR/WCAG21/#dfn-pointer-inputs) to be at least **44 by 44 in CSS pixels**. Meeting this requirement has [multiple benefits](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html#benefits), it'd be great to pass this criteria for the commonly used controls:

- General controls:
  https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L284-L286<br>
- Markers:
  https://github.com/Leaflet/Leaflet/blob/bc918d4bdc2ba189807bc207c77080fb41ecc196/src/layer/marker/Icon.Default.js#L26
  (Changing the marker's `width` and `height` to `44px` would cause it to stretch. Either the image would need to be modified or `padding` (right and left) could be modified to achieve a 44 by 44 px target.)<br><br>
- Popup's Close button:
  https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L517-L518
  (Here too `padding` can be utilized if keeping a visually small icon is desired.)<br><br>

Increasing the default size of controls would also have the benefit of not needing extra CSS for conditional sizing (presumably, since the controls would be large enough by default). For example, the following lines could be removed:

https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L316-L320

https://github.com/Leaflet/Leaflet/blob/0f904a515879fcd08f69b7f51799ee7f18f23fd8/dist/leaflet.css#L359-L362

**Comment by Malvoz** [comment]
In any case 44 by 44 px cannot be achieved for these targets (for example, for markers it can get tricky if increasing the size of the image itself is not desired, but setting a `padding` is also not desired as it may cause markers to be unexpectedly interacted with since they're larger than percieved) it should be added to as a known limitation in the [accessibility statement](https://github.com/Leaflet/Leaflet/issues/7537).
